hudi-common/src/main/avro/HoodieMetadata.avsc

```diff
@@ -417,6 +417,35 @@
                             "name": "isTightBound",
                             "type": "boolean",
                             "default": false
+                        },
+                        {
+                            "doc": "Value type information containing enum ordinal and optional additional information",
```

Comment:
Given that the min and max values stored in column stats are changed and versioned, could you check how XTable can use the same utils or new functionality for reading column stats from Hudi in a backward compatible way?

Reply:
ValueMetadata has several static methods `getValueMetadata` that can be used on avro schema, avro parquet type,  or `HoodieValueTypeInfo` contained in a HoodieMetadata col stats record. 

https://github.com/apache/incubator-xtable/blob/9f989bf1988bf9ce29f66bd2a9fe04bb4db73c6a/xtable-api/src/main/java/org/apache/xtable/model/stat/Range.java#L60
The docs here say that timestamps are using some internal type. So seems like some conversion will be necessary in xtable. 


And the utils here need to be updated:
https://github.com/apache/incubator-xtable/blob/main/xtable-core/src/main/java/org/apache/xtable/hudi/HudiFileStatsExtractor.java